test_unstructured/partition/test_api.py

```diff
@@ -102,7 +102,7 @@ def test_partition_via_api_raises_with_bad_response(request: FixtureRequest):
     partition_mock_.assert_called_once()
 
 
-@pytest.mark.skipif(skip_outside_ci, reason="Skipping test run outside of CI")
+@pytest.mark.skipif(not is_in_ci, reason="Skipping test run outside of CI")
```

Comment:
Are the rest of these skipped for the same reason?

Reply:
I don't know. I didn't look into it